app/forms/state_file/nc_subtractions_form.rb

```diff
@@ -0,0 +1,27 @@
+module StateFile
+  class NcSubtractionsForm < QuestionsForm
+    set_attributes_for :intake, :tribal_member, :tribal_wages_amount
+
+    validates :tribal_member, inclusion: { in: %w[yes no], message: :blank }
+    validates_numericality_of :tribal_wages_amount, if: -> { tribal_member == "yes" }
+    validates :tribal_wages_amount, presence: true, allow_blank: false, numericality: { greater_than_or_equal_to: 1 }, if: -> { tribal_member == "yes" }
```

Comment:
i wonder if `validates_numericality_of` is duplicative of `numericality` check on the next line? but I see this is the pattern for other amount fields, so maybe I'm missing something

Reply:
good call this is redundant! im fixing this